src/adaptation/icap/Xaction.cc

```diff
@@ -134,6 +134,22 @@ void Adaptation::Icap::Xaction::start()
     Adaptation::Initiate::start();
 }
 
+// TODO: Make reusable by moving this (and the printing operator from
+// ip/Address.h that this code is calling) into ip/print.h or similar.
```

Comment:
I marked this PR as a Draft because I was not sure whether this TODO should be addressed in this unofficial PR, in the corresponding official PR, or later, in a followup official PR. Your call.